e2e/page-objects/sign-in-page.ts

```diff
@@ -0,0 +1,32 @@
+import { Page } from '@playwright/test';
+import { expect } from 'playwright/test';
+import { HOMEPAGE_ENDPOINT, SIGN_IN_PAGE_ENDPOINT } from '../constants';
+
+export class SignInPage {
+  readonly page: Page;
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+
+  emailInput = () => this.page.locator('#email');
+  passwordInput = () => this.page.locator('#password');
+  signInButton = () => this.page.getByRole('button', { name: 'Sign in', exact: true });
+  signInNavButton = () => this.page.getByText('Sign in');
+  signUpButton = () => this.page.locator('a').getByText('Sign up', { exact: true });
+
+  async verifyUserIsSignedIn() {
+    await expect(this.signInNavButton()).toHaveCount(0);
+    await expect(await this.page.getByTestId('account-icon').locator('a').getAttribute('href')).toBe('/account');
+  }
+
+  async signIn({ email, password }: { email: string; password: string }) {
+    await this.page.goto(SIGN_IN_PAGE_ENDPOINT);
+    await this.emailInput().fill(email);
+    await this.passwordInput().fill(password);
+    await this.signInButton().click();
+    await this.page.waitForTimeout(2000);
```

Comment:
For any timeouts a comment that explains why it's necessary would be helpful, and link to a bug if applicable